lib/core/Container.js

```diff
@@ -36,7 +36,7 @@ class Container extends React.Component {
 }
 
 Container.defaultProps = {
-  background: 'transparent',
+  background: null,
```

Comment:
The default value (`transparent`) was not being used, so it makes no difference setting it to `null`.